### ISSUE TITLE
nvidia patch error

### ISSUE THREAD

**Issue by linmuqin** [keylase/nvidia-patch#664]
Hello,
I get the attached offset error when trying to apply nvencodeapi64.1337 to nvEncodeAPI64.dll.Already restarted and it doesn't work
Please help me
![image](https://github.com/keylase/nvidia-patch/assets/56988241/cc3c9aef-79b1-4ddf-baf0-b2a1992bf49d)

Graphics card driver version: 512.15
Graphics card model: NVIDIA GeForce RTX 3060
The attachments are nvEncodeAPI64.dll and nvencodeapi64.1337 files

**Comment by jailuthra** [comment]
From the error looks like you're trying to patch an already patched driver?

Can you try testing if your session limit is already removed? https://github.com/keylase/nvidia-patch/wiki/Verify-NVENC-patch

**Comment by linmuqin** [comment]
Thank you for your reply, I downloaded and placed it in any directory, but an error was reported when I ran it. Do you want to place it in the same directory as the nvidia-patch file?
![image](https://github.com/keylase/nvidia-patch/assets/56988241/4c429bc9-e9f3-4475-9454-cf5720efe6e0)

**Comment by jailuthra** [comment]
Hmm no need it should run from anywhere. Looks like the steps to test in wiki are out of date as something changed in ffmpeg.

Will test it on my machine later.

**Comment by jailuthra** [comment]
Oh wait maybe it's your powershell that is causing issue with line breaks. @linmuqin can you try the following script instead https://github.com/keylase/nvidia-patch/raw/master/win/tools/patch-tester/patch-tester.ps1

**Comment by linmuqin** [comment]
It runs successfully, but the result is wrong
![image](https://github.com/keylase/nvidia-patch/assets/56988241/1f636cb2-15c8-4df7-8e93-6cf8a16ffa0d)

**Comment by linmuqin** [comment]
My graphics card driver version and the files used are attached, thank you for your contribution
![image](https://github.com/keylase/nvidia-patch/assets/56988241/ef7d452a-3613-4a4d-9367-2c34965b3303)
[download.zip](https://github.com/keylase/nvidia-patch/files/11752878/download.zip)

**Comment by linmuqin** [comment]
I actually had it patched before and recently changed the graphics card. Do I need to apply the patch again, does the previous one still work?

**Comment by jailuthra** [comment]
@linmuqin Yeah looks like your driver is too old for latest ffmpeg.

But if you patched driver before, and only changed graphics card and did not update driver, it should still be patched.

You can just try using it for whatever purpose you were using before (encoding or framecapture) - if you see any issues around session limit maybe then try to reinstall driver and patch again.

**Comment by linmuqin** [comment]
Yes, when I started coding it gave an error, I guess it needs to be patched again as soon as the video card is changed. A workaround I found was to reinstall the graphics driver so I could patch again.

**Comment by jailuthra** [comment]
there's no expiry unless you update or reinstall the driver.

**Comment by linmuqin** [comment]
Awesome, thank you so much for your patience
